2-TokenCache/Console-TokenCache/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Instance": "https://login.microsoftonline.com/",
+  "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
```

Comment:
This is a public client application. Don't we want to provide a clientID ?

Reply:
Yes, let's preconfigure this app!

Reply:
In an agreement with Kalyan, we wont preconfigure the app. We received a recommendation to avoid hardcoding clientIds. We can discuss this topic internally if needed.

Reply:
Please start an email thread on this Tiago if you do not wish to discuss this in public (I think we should). Personally I find precofigured apps easier to use. A lot of our samples come preconfigured.

Also, it can take a really long time for people to configure apps, if they are not familiar with AAD.